<launch> 
    <node name="g3t1_1" pkg="component" type="g3t1_1" output="screen" />

    <param name="frequency" value="0.5" /> <!-- 1 Hz  -->
    <param name="deadline" value="2000000" /> <!-- (ns) deadline = 10^6*(1/frequency)-->
    <param name="wce" value="1000" /> <!-- 1000Î¼s -->
    <param name="check_frequency" value="10" /> <!-- 10 Hz -->

    <!-- Defines the percentages to consider low, moderate or high risk -->
    <param name="lowrisk" value="0,20" />
    <param name="midrisk" value="21,65" />
    <param name="highrisk" value="66,100" />

    <!-- markov chain -->
    <param name="state0" value="0,0,0,0,0" />
    <param name="state1" value="0,0,0,0,0" />
    <param name="state2" value="0,0,20,20,60" />
    <param name="state3" value="0,0,20,20,60" />
    <param name="state4" value="0,0,20,20,60" />

    <!-- risk values -->
    <param name="LowRisk" value="80,100" />
    <param name="MidRisk1" value="60,80" />
    <param name="HighRisk1" value="0,60" />

    <!-- accuracy in percentage -->
    <param name="accuracy" value="90" type="double" />

</launch>